docs/troubleshooting-guide.md

```diff
@@ -0,0 +1,146 @@
+# Front-End Monorepo - Troubleshooting Guide
+
+At this moment in time - the distant past of October 2018 - the Zooniverse
+Front-End Monorepo is a rapidly fluctuating work in progress. New devs jumping
+in may have trouble figuring out how to even get started, so this evolving
+document was added to help with the weird errors and issues that may not yet be
+covered in the "proper" documentation.
+
+(@shaun.a.noordin 2018.10.12)
+
+NOTE: this evolving doc should be reviewed regularly and should be retired
+eventually. Any important points should be folded into the Monorepo's proper
+documentation.
+
+## Installing & Running Packages
+
+Do what the README.MD tells you:
+
+1. Clone the repo: `git clone git@github.com:zooniverse/front-end-monorepo.git`
+2. Run Lerna's bootstrap: `lerna bootstrap`
+  1. NOTE: this installs dependencies for every package, but be prepared for
+     jankiness as memory leaks might occur, files are opened for writing but
+     never closed, the OS lagging and etc. Doesn't always happen, but sometimes
+     it does.
+  2. NOTE: Despite the README saying that you should first enter into the
+     package directory before running bootstrap (`cd packages/whatever`), it
+     seems that Lerna will bootstrap _all_ packages regardless of which
+     directory you're in.
+3. Go to the package that you're interested in: `cd packages/whatever`
+4. In the package folder, you MAY want to install the dependencies manually:
```

Comment:
In theory `lerna bootstrap` should be installing whatever dependencies that are defined in the package.json if you run this command within the package directly. Ideally you shouldn't be running `npm install` directly as this can break if there are symlinked packages being managed by lerna. However, in practice, I sometimes have to do this.